zebra/zserv.c

```diff
@@ -573,8 +573,10 @@ static void zserv_process_messages(struct event *thread)
 int zserv_send_message(struct zserv *client, struct stream *msg)
```

Comment:
could we go further here - could this actually just drop all the messages except for the couple that are used by sync connections?
or could we have sync connections specify the handful of messages they will process, and maintain that list on the zebra side?